ModularTegustation/tegufood.dm

```diff
@@ -22,3 +22,48 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 10)
 	tastes = list("stale bread" = 1)
 	foodtypes = GRAIN
+
+	//LC13
+/datum/crafting_recipe/food/pbirdcookie
+	time = 15
+	name = "Punishing Bird Cookie"
+	reqs = list(
+		/datum/reagent/consumable/sugar = 5,
+		/obj/item/food/pastrybase = 1
+	)
+	result = /obj/item/food/cookie/sugar/pbird
+	subcategory = CAT_PASTRY
+
+/obj/item/food/cookie/sugar/pbird
+	icon_state = "sugarcookie_pbird"
+
+/datum/crafting_recipe/food/wormfood
+	name = "Wormfood"
+	reqs = list(
+		/obj/item/food/meat/slab/worm = 1,
+		/obj/item/food/cheesewedge = 3
+	)
+	result = /obj/item/food/wormfood_fail
+	subcategory = CAT_MEAT
+
+/obj/item/food/wormfood_fail
+	name = "wormfood"
+	desc = "Something inside the meat is desperately consuming whatever is left."
+	icon_state = "wormfood1"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/drug/maint/tar = 1, /datum/reagent/yuck = 2)
+	tastes = list("crunchy popping" = 1, "consumption" = 1)
+	foodtypes = MEAT | GROSS
+
+/obj/item/food/wormfood/Initialize()
+	. = ..()
+	if(prob(50))
+		new /obj/item/food/wormfood(get_turf(src))
+		qdel(src)
```

Comment:
I think you meant to use another type in either definition or when creating new food.